agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -129,13 +141,14 @@ function collect_sos {
 	debug_log "[${script_name}]collecting sosreport"
 	${_cmd} > ${dir}/sosreport-${_name}.log 2>&1
 
-	# The latest version of sosreport (3.6 right now) generates different names
-	# for the sosreport tar balls, inserting a short hostname before the label.
-	# Since older versions of sosreport did not do that, we use a wildcard
-	# pattern to capture both names when checking for success.
+	# Version of sosreport >= 3.6 generate different names for the
```

Comment:
If we want this to be perfect, then

s/Version/Versions/

but I'm not going to quibble.

Reply:
Sigh ... I'll leave it as is for now.